<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vdlSettingsDialog</class>
 <widget class="QDialog" name="vdlSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>160</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>510</width>
    <height>160</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>510</width>
    <height>160</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../ExMplayer.qrc">
    <normaloff>:/images/ExMplayer.png</normaloff>:/images/ExMplayer.png</iconset>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>110</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>54</y>
     <width>491</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>youtube-dl Path :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditYoudlDir"/>
    </item>
    <item>
     <widget class="QToolButton" name="toolButtonBrFolderYdl">
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>20</y>
     <width>491</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Download Video/Audio to  Folder :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditDomDir"/>
    </item>
    <item>
     <widget class="QToolButton" name="toolButtonBrFolder">
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButtonReset">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonUpdate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Update youtube-dl</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../ExMplayer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>vdlSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>vdlSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
